src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -688,7 +688,7 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
         // otherwise, just query subject index for subject_id list,
         // and query file index for manifest info.
```

Comment:
```suggestion
        // and query file index for manifest info.
        // Setting downloadingInProgress of manifest to true as a trick such that the button's isPending
        // state is set to true while the manifest is being counted. This is because we want the button to show
        // the spinner instead of the download icon while the counting is in progress. 
```

Reply:
Added some comments to make it clear for people who work on this in the future.